src/test/java/cat/udl/eps/softarch/academicrecruit/domain/CandidateTest.java

```diff
@@ -0,0 +1,24 @@
+package cat.udl.eps.softarch.academicrecruit.domain;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class CandidateTest {
```

Comment:
This class doesn't seem necessary, specially to test methods that are actually generated by external code (Lombok)